package.json

```diff
@@ -1,7 +1,7 @@
 {
   "author": "OpusCapita (www.opuscapita.com)",
   "name": "oc-common-ui",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
Do not update the version manually: the command 'npm version [x]' does this automatically